### ISSUE TITLE
Change popover implementation from Bootstrap to Yale's

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble#39]
> During user testing, tester A could not tell that green underlined words throughout the interview were interactive: "I don't even know those exist". He called it "definitely a blocker". Tester B (VoiceOver, iPhone) also could not access the tooltip content when directed to them by the moderator. The definition terms use Bootstrap popovers (https://getbootstrap.com/docs/5.3/components/popovers/) to display definitions, but the popover content is not announced to screen readers when the element is activated or focused. During the expert audit, VoiceOver on macOS was able to focus the element via Tab but did not automatically announce the popover content. The content was accessible through VoiceOver's "more content" shortcut, but this depends on user verbosity settings and is not a reliable way to surface essential information

> Review Yale's accessible popover implementation (https://yale-a11y.gitlab.io/ui-component-library/tooltips-popovers), which uses a dismissible popover pattern that works with VoiceOver. As a longer-term fix, consider replacing the popover with the native HTML "<details>" and "<summary>" elements (https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/details), which provide a built-in accessible disclosure pattern without JavaScript. Also consider Yale's guidance notes that popovers "should never contain essential content," and definition terms on a legal form are essential content.

(See A11Y-06)
